README.md

```diff
@@ -56,4 +73,75 @@ with the optimizer is
 optimizer_version = '0.12.9'
```

Comment:
0.12.11, rust 1.66

Reply:
Yes, I update optimizer_version to 0.12.11.
For Rust version already mentioned in line 48: "You will need Rust 1.66.0+ with wasm32-unknown-unknown target installed."